src/database/rrd.c

```diff
@@ -147,6 +147,11 @@ int rrd_init(const char *hostname, struct rrdhost_system_info *system_info, bool
     if (unlikely(!localhost))
         return 1;
 
+    char default_port_str[16];
+    uint16_t default_port = (web_server_mode == WEB_SERVER_MODE_NONE) ? 0 : web_server_get_default_port();
```

Comment:
This is wrong if `bind to` port != `default port`.


With the following config:

```
[web]
	bind to = 10.10.10.20:19997 10.10.10.20:19998
```

I get

```
"host_labels": {
"_web_listen_port": "19999",
```

Is that easy to fix? If yes, let's do it now. Otherwise, let's add TODO.